### DISCUSSION TITLE
HW 4 part 2 problem 2c

### DISCUSSION THREAD

**Discussion by Aarons-Joshua** [cooperrc/computational-mechanics#57] [Q&A]
Does anyone know how to first calculate the P1 for the given forces in this problem. I am having trouble with the code and plotting as I cannot pull out the values for the p1 that I calculated. Here is my code:

![image](https://user-images.githubusercontent.com/98834673/175545435-45e8a82d-6efe-4a58-9e81-7b1dfc5640b4.png)

**Comment by ecieszynski** [comment]
it should be in the x vector you get from the gaussian elimination code right?

**Reply by cooperrc** [reply]
Exactly, take a look at your original equations that you defined as $Ax=y$

$x = [P_1,~P_2,~R_2]$
